RiotSharp/Interfaces/IRiotApi.cs

```diff
@@ -20,6 +20,7 @@ namespace RiotSharp.Interfaces
     public interface IRiotApi
     {
         #region Summoner
+
```

Comment:
One little thing, before I'll check tomorrow: Can you remove these new line after a region start end. When I introduced these regions, I tried to keep the code as compact as possible.

Reply:
Okay, I'm going to fix that. (And I'm going to kill Resharper)

Reply:
nit: I think the problem has reappeared with the rebase?